src/components/text/index.tsx

```diff
@@ -1,19 +1,7 @@
 import React, { FC } from 'react';
 import { Text as ChakraText, TextProps } from '@chakra-ui/react';
 
-type Props = Pick<
-  TextProps,
-  | 'as'
-  | 'children'
-  | 'color'
-  | 'textStyle'
-  | 'textAlign'
-  | 'isTruncated'
-  | 'noOfLines'
-  | 'marginBottom'
->;
-
-const Text: FC<Props> = ({ children, ...chakraProps }) => (
+const Text: FC<TextProps> = ({ children, ...chakraProps }) => (
```

Comment:
We can just import and export Chakra Text Comp if all props are enabled.

Reply:
True, true, true! Thanks!